build.sbt

```diff
@@ -55,7 +55,8 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     buildInfoSettings,
     libraryDependencies ++= Seq(
       metaconfig.value,
-      scalameta.value
+      scalameta.value,
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
```

Comment:
Can we add a comment explaining why this is needed? 

> scala-reflect is an undeclared dependency of fansi, see https://github.com/scalameta/scalafmt/issues/1252. Scalafmt itself does not require scala-reflect.

It's a big dependency and it would be great to avoid it in the future.

Reply:
Sure! Added https://github.com/scalameta/scalafmt/pull/1284/commits/2a13a4b01301f1ccc86381b8ec3b83587f8e79a1

> It's a big dependency and it would be great to avoid it in the future.

I agree.